src/security/KeyData.cc

```diff
@@ -84,58 +122,6 @@ Security::KeyData::loadX509CertFromFile()
     return bool(cert);
 }
 
-/// load any intermediate certs that form the chain with the loaded signing cert
-void
-Security::KeyData::loadX509ChainFromFile()
```

Comment:
I could preserve this function (instead of moving/inlining it). Doing so would make the PR diff even smaller, but I would have to give this function a BIO reference as a parameter, and that would make its profile OpenSSL-specific. Also, it would, IMO, make comprehending this code _more_ difficult: The temporary try/catch noise aside, the code became easier to read once all certificate loading code was merged together in side loadCertificates().

Said that, if others insist on restoring this function (with a BIO parameter), I will do that. Just let me know.

Reply:
IIRC the original idea behind this separation was to allow for loading files defined by other port settings (eg tls-cafile=,  clientca=, or capath= ) without having it locked to the tls-cert= file. But it ended up getting locked anyway with internal use of the certFile member.

Reply:
We can always extract this code into a method again if it becomes useful to do so. The current PR code is best for the current use case, and it does not create any serious lock-in AFAICT.